
global_frame: odom
robot_base_frame: base_link
update_frequency: 5.0
publish_frequency: 2.0
width: 3.0  # Smaller window for local
height: 3.0
resolution: 0.05
rolling_window: true
transform_tolerance: 0.5


# # Remove origin_x/y for rolling window

plugins:
  - {name: static_layer, type: "costmap_2d::StaticLayer"}
  - {name: inflation_layer, type: "costmap_2d::InflationLayer"}


# plugins:
#   - {name: obstacle_layer, type: "costmap_2d::ObstacleLayer"}
#   - {name: inflation_layer, type: "costmap_2d::InflationLayer"}


static_layer:
  map_topic: /semantic_costmap
  subscribe_to_updates: true
  track_unknown_space: true
  combination_method: 1


# global_frame: odom
# robot_base_frame: base_link
# update_frequency: 10.0
# publish_frequency: 2.0
# width: 6.0
# height: 6.0
# resolution: 0.05
# rolling_window: true
# transform_tolerance: 0.5

# plugins:
#   - {name: static_layer, type: "costmap_2d::StaticLayer"}
#   - {name: inflation_layer, type: "costmap_2d::InflationLayer"}

# static_layer:
#   map_topic: /semantic_costmap
#   subscribe_to_updates: true
#   track_unknown_space: true
#   combination_method: 1
#   enabled: true

# inflation_layer:
#   inflation_radius: 0.25
#   cost_scaling_factor: 3.0
